packages/mdc-list/_evolution-mixins.scss

```diff
@@ -696,8 +696,7 @@
   @include feature-targeting.targets($feat-structure) {
     // Extra specificity needed to override default customizations.
     &.mdc-list-item {
-      // We rely on auto to avoid clobbering the opposite padding.
-      @include rtl-mixins.reflexive-property(padding, 0, auto);
+      @include rtl-mixins.reflexive-property(padding, 0, 16px);
```

Comment:
Thank you so much for investigating this issue -- we definitely want to get RTL padding straightened out. Unfortunately, setting an explicit padding here won't work with how we've implemented leading and trailing variants.

The use of "auto" is an intentional workaround to get fall-through behavior so that we can set leading and trailing padding independently. By passing auto, we're able to clear the leading padding without influencing the trailing padding ("auto" is ignored and the previous value is retained). 

This allows us to have each leading and trailing style to be independent; if we didn't use this technique, we'd need to style every possible combination of leading/trailing variant independently (since the only cross-browser method of specifying RTL/LTR styles requires both left/right values to be specified simultaneously).

That said, we should definitely figure out a fix for the leading padding being lost in RTL. I'll investigate this on our end and update #7024. For now, it may be best to remove this commit from the PR.

Reply:
Oh I see. I started thinking about the possible combinations of each leading/trailing variant, but then just naively though using `16px` as elsewhere, would be sufficient. The problem with `auto` is that it will fall through, but only to the LTR value.. so it ends up with the `0px`. I can't think of a simple solution to this, but the ideal one would be to only apply the LTR value if the browser is not setting `dir=rtl` (though that has other downsides)

Reply:
@brandondiamond Removed the commit. thanks for reviewing.